components/brave_new_tab_ui/components/default/cryptoDotCom/index.tsx

```diff
@@ -0,0 +1,424 @@
+          return (
+            <ListItem key={currency} isFlex={true} onClick={this.handleAssetDetailClick.bind(this, currency)} style={{ height: 48 }}>
+              <FlexItem style={{ paddingLeft: 5, paddingRight: 5 }}>
+                {this.renderIconAsset(currency.toLowerCase())}
+              </FlexItem>
+              <FlexItem>
+                <Text>{currency}</Text>
+                <Text small={true} $color='light'>{currencyNames[currency]}</Text>
+              </FlexItem>
+              <FlexItem textAlign='right' flex={1}>
+                {(price !== null) && <Text>{new Intl.NumberFormat('en-IN', { style: 'currency', currency: 'USD', currencyDisplay: 'narrowSymbol' }).format(price)}</Text>}
+                {(percentChange !== null) && <Text $color={percentChange > 0 ? 'green' : 'red'}>{percentChange}%</Text>}
+              </FlexItem>
+            </ListItem>
+          )
+        })}
+      </List>
+    )
+  }
+
+  renderAssetDetailView () {
+    const { selectedAsset: currency } = this.state
+    const { price = null, volume = null } = this.props.tickerPrices[currency] || {}
+    const chartData = this.props.charts[currency] || []
+    const pairs = this.props.supportedPairs[currency] || []
+
+    const losersGainers = this.transformLosersGainers(this.props.losersGainers || {})
+    const { percentChange = null } = losersGainers[currency] || {}
+
+    const chartHeight = 100
+    const chartWidth = 309
+    return (
+      <Box noPadding={true}>
+        <FlexItem
+          hasPadding={true}
+          style={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+            width: '100%',
+            borderBottom: '1px solid rgba(79, 86, 97, 0.7)'
+          }}
+        >
+          <FlexItem>
+            <BackArrow>
+              <CaratLeftIcon onClick={this.setSelectedAsset.bind(this, '')} />
+            </BackArrow>
+          </FlexItem>
+          <FlexItem style={{ paddingRight: 5 }}>
+            {this.renderIconAsset(currency.toLowerCase())}
+          </FlexItem>
+          <FlexItem flex={1}>
+            <Text>{currency}</Text>
+            <Text small={true} $color='light'>
+              {currencyNames[currency]}
+            </Text>
+          </FlexItem>
+          <FlexItem style={{ paddingLeft: 5 }}>
+            <ActionButton small={true} light={true}>
+              BUY
+            </ActionButton>
+          </FlexItem>
+        </FlexItem>
+        <FlexItem
+          hasPadding={true}
+          style={{
+            width: '100%',
+            borderBottom: '1px solid rgba(79, 86, 97, 0.7)'
+          }}
+        >
+          {(price !== null) && <Text
+            inline={true}
+            large={true}
+            weight={500}
+            style={{ marginRight: '0.5rem' }}
+          >
+            {new Intl.NumberFormat('en-IN', { style: 'currency', currency: 'USD', currencyDisplay: 'narrowSymbol' }).format(price)} USDT
```

Comment:
For instances where we're using `Intl.NumberFormat`, can we create a reusable method within in the class for this? ex: `{this.getFormattedPrice(price)}`